components/shared/main-nav.js

```diff
@@ -10,11 +10,15 @@ const styles = {
     root: {
         flexGrow: 1,
         backgroundColor: '#003da5',
+        margin: '0 auto',
+        width: '100%',
     },
     navBar: {
         boxShadow: 'none!important',
-        maxWidth: 1200,
+        Width: '100%',
         margin: '0 auto',
+        position: 'fixed',
+        paddingLeft: '17%',
```

Comment:
Reason for `paddingLeft: '17%',` ? Seems like a really high percentage for it to be at.
It looks fine at 1600px width, but odd at every other size.